stripe/src/main/java/com/stripe/android/model/Card.java

```diff
@@ -182,14 +186,65 @@ public Card(String number, Integer expMonth, Integer expYear, String cvc, String
         this.currency = TextUtils.nullIfBlank(currency);
     }
 
-    public Card(String number, Integer expMonth, Integer expYear, String cvc, String name, String addressLine1, String addressLine2, String addressCity, String addressState, String addressZip, String addressCountry, String currency) {
-        this(number, expMonth, expYear, cvc, name, addressLine1, addressLine2, addressCity, addressState, addressZip, addressCountry, null, null, null, null);
-    }
-
-    public Card(String number, Integer expMonth, Integer expYear, String cvc) {
-        this(number, expMonth, expYear, cvc, null, null, null, null, null, null, null, null, null, null, null);
-    }
-
+    public Card(
+            String number,
+            Integer expMonth,
+            Integer expYear,
+            String cvc,
+            String name,
+            String addressLine1,
+            String addressLine2,
+            String addressCity,
+            String addressState,
+            String addressZip,
+            String addressCountry,
+            String currency) {
+        this(
+                number,
+                expMonth,
+                expYear,
+                cvc,
+                name,
+                addressLine1,
+                addressLine2,
+                addressCity,
+                addressState,
+                addressZip,
+                addressCountry,
+                null,
+                null,
+                null,
+                null);
+    }
+
+    public Card(
```

Comment:
I have my doubts that these constructors are useful, but as long as they're here, one argument per line.